cordformation/src/main/kotlin/net/corda/plugins/Node.kt

```diff
@@ -470,6 +491,50 @@ open class Node @Inject constructor(private val project: Project) {
         }
     }
 
+
+    private fun createSchemasCmd() = listOfNotNull(
+            Paths.get(System.getProperty("java.home"), "bin", "java").toString(),
+            "-jar",
+            "corda.jar",
+            "run-migration-scripts",
+            if (allowHibernateToManageAppSchema.get()) "--allow-hibernate-to-manage-app-schema" else null)
+
+    private fun runSchemaMigration(){
+        if (!runSchemaMigration.get()) return
+        project.logger.lifecycle("Run database schema migration scripts${if(allowHibernateToManageAppSchema.get()) " - managing CorDapp schemas with hibernate" else ""}")
+        runNodeJob(createSchemasCmd(), "node-schema-cordform.log")
+    }
+
+    val LOGS_DIR_NAME: String = "logs"
```

Comment:
This property should probably be `private`, or else Gradle will expect it to be annotated with `@Input`, `@Internal`etc.